packages/manager/modules/pci/src/projects/project/storages/databases/fork/fork.module.js

```diff
@@ -2,6 +2,7 @@ import angular from 'angular';
 
 import '@ovh-ux/ng-ovh-cloud-universe-components';
 import '@ovh-ux/ng-ovh-swimming-poll';
+import '@ovh-ux/ng-ovh-utils';
```

Comment:
**nitpick:** Importing `@ovh-ux/ng-ovh-utils` has a significant cost.
see: https://bundlephobia.com/package/@ovh-ux/ng-ovh-utils@14.0.14

Furthermore, we are deprecating this module so I would recommend you
to take a closer look to following dependency:

- `@ovh-ux/ng-ovh-cloud-universe-components`

As this dependency is already imported, you will be able to user following filter `cucBytes`.

Please let me know if its cover well your needs.
Thanks

Reply:
Works like a charm :)
On a sidenote, is there a way to know if modules are gonna be deprecated ?

Reply:
That's a good point.
For now, we tend to update `README.md` file per package or we just update their `package.json`(description field).
We will improve this to get a better overview of what's its deprecated.
Thanks